cmd/boost/index.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/fatih/color"
+	bcli "github.com/filecoin-project/boost/cli"
+	lcli "github.com/filecoin-project/lotus/cli"
+	"github.com/urfave/cli/v2"
+)
+
+var indexProvCmd = &cli.Command{
+	Name:  "index",
+	Usage: "Manage the index provider on Boost",
+	Subcommands: []*cli.Command{
+		indexProvAnnounceAllCmd,
+	},
+}
+
+var indexProvAnnounceAllCmd = &cli.Command{
+	Name:  "announce-all",
+	Usage: "Announce all active deals to indexers so they can download the indices",
+	Flags: []cli.Flag{
+		&cli.BoolFlag{
+			Name:        "color",
+			Usage:       "use color in display output",
+			DefaultText: "depends on output being a TTY",
+		},
+	},
+	Action: func(cctx *cli.Context) error {
+		if cctx.IsSet("color") {
+			color.NoColor = !cctx.Bool("color")
+		}
```

Comment:
This should probably be a flag at the global level (not per command).
I'd say let's add it in a later PR if we need it.